c-api/dotnet/src/Picky/Native/Raw.cs

```diff
@@ -0,0 +1,59 @@
+using System.Runtime.InteropServices;
```

Comment:
Question: this is generated?

Reply:
Yes everything in the `Native` folder is generated